internal/cephfs/nodeserver.go

```diff
@@ -181,6 +184,18 @@ func (*NodeServer) mount(ctx context.Context, volOptions *volumeOptions, req *cs
 			err)
 		return status.Error(codes.Internal, err.Error())
 	}
+	if !csicommon.MountOptionContains(kernelMountOptions, readOnly) && !csicommon.MountOptionContains(fuseMountOptions, readOnly) {
+		// #nosec - allow anyone to write inside the stagingtarget path
+		err = os.Chmod(stagingTargetPath, 0777)
```

Comment:
Should this be done recursive for all the contents? I guess it was not the case, but for creating a volume from a snapshot or other pvc, it may be needed.

Reply:
This is needed to provide write access to normal users

Reply:
Yes, I understand that, but the current command assumes there is no content on the volume that needs changes in permissions. In case contents is there (cloned volume), the `chmod` should probably change the permissions of everything on it.

Reply:
i hope same is going to happen without this change also, We need to have a separate issue to handle it can you please open one?

Reply:
#1183 has been opened for this